src/H4LTools.cc

```diff
@@ -1055,23 +1055,23 @@ bool H4LTools::GetZ1_2l2qOR2l2nu()
 ////// emu control region (Z1 candidate selection) /////////
 bool H4LTools::GetZ1_emuCR()
 {
-    if (DEBUG)
-        std::cout << "##Inside function GetZ1_emuCR()" << std::endl;
+    //if (DEBUG)
```

Comment:
Don't comment `if (DEBUG)`.

You can make it true or false from post_proc.py

https://github.com/ram1123/nanoAOD_skim/blob/6d2833f3dc3b3e69e956e8bc22cfcd5ced6fdc2a/post_proc.py#L31